service/dap/server.go

```diff
@@ -1555,6 +1562,64 @@ func (s *Server) onCancelRequest(request *dap.CancelRequest) {
 	s.sendNotYetImplementedErrorResponse(request.Request)
 }
 
+// onExceptionInfoRequest handles 'exceptionInfo' requests.
+// Capability 'supportsExceptionInfoRequest' is set 'initialize' response.
+func (s *Server) onExceptionInfoRequest(request *dap.ExceptionInfoRequest) {
+	goroutineID := request.Arguments.ThreadId
+	var body dap.ExceptionInfoResponseBody
+	// Get the goroutine and the current state.
+	g, _ := s.debugger.FindGoroutine(goroutineID)
+	state, _ := s.debugger.State( /*nowait*/ true)
+	var bpState *proc.BreakpointState
+	if g != nil && g.Thread != nil {
+		// Check if this goroutine ID is stopped at a breakpoint.
+		bpState = g.Thread.Breakpoint()
+	}
+	if bpState != nil && bpState.Breakpoint != nil {
+		switch bpState.Breakpoint.Name {
+		case proc.FatalThrow:
```

Comment:
@aarzilli Why isn't there a special stop reason for these?

Reply:
The way I see it, they are normal breakpoints.